vertexai/VertexAISnippets/VertexAISnippets.swift

```diff
@@ -381,7 +381,7 @@ class Snippets {
     // Initialize the generative model
     // Use a model that supports function calling, like Gemini 1.0 Pro.
```

Comment:
Would you mind changing that text to "like a Gemini 1.5 model"? It looks like we've gotten out of sync with the latest iteration of the [docs](https://firebase.google.com/docs/vertex-ai/function-calling?platform=ios#specify-function-declaration-during-model-initialization).

Reply:
Done.